libc/src/__support/FPUtil/arm/FEnvImpl.h

```diff
@@ -11,7 +11,6 @@
 
 #include "src/__support/FPUtil/FPBits.h"
```

Comment:
Add #include "hdr/fenv_macros.h here also. It seems redundant, but it will make the next step easier.